djlint/settings.py

```diff
@@ -1123,3 +1123,9 @@ def __init__(
               )
         """
         )
+
+    def get_resolved_exclude_paths(self) -> list[Path]:
+        exclude_paths = []
+        for p in self.exclude.split("|"):
+            exclude_paths.append(Path(p.strip().replace("\\.", ".")).resolve())
```

Comment:
I did not want to change the way [how `default_exclude` are passed]( https://github.com/djlint/djLint/blob/master/djlint/settings.py#L478) so I had to tweak it a bit to return a list of resolved `Paths` here 🙇🏻